Tasks/Xcode/task.json

```diff
@@ -213,8 +213,17 @@
             "required":false,
             "helpMarkDown": "Test reporter format to use when \"test\" action is specified and \"Use xctool\" is checked. Specify \"junit:output-file-path-here.xml\" to generate a file format compatible with the Publish Test Results task. When specified, \"plain\" is automatically added as well. Requires xctool be installed on agent hosts. See [xctool](https://github.com/facebook/xctool) for details.",
             "groupName":"advanced"
+        },
+		{
+            "name":"publishJUnitResults",
+            "type":"boolean",
+            "label":"Publish to VSO/TFS",
```

Comment:
change to just 'VSTS'.  We will be renaming but we shouldn't be adding new VSO references